index.html

```diff
@@ -5,7 +5,8 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Vite App</title>
+  <link href="public/images/favicon.ico" rel="icon" type="image/x-icon" />
```

Comment:
Tą ścieżka pewnie nie będzie działać. Powinno być:
```suggestion
  <link href="/images/favicon.ico" rel="icon" type="image/x-icon" />
```

Reply:
Zerknij tam pod tym linkiem co na discordzie napisałem czy jest ta favicona i daj znać jak możesz bo ja na telefonie nie widzę :D